radio/src/keys.cpp

```diff
@@ -41,10 +41,46 @@
 event_t s_evt;
 struct InactivityData inactivity = {0};
 Key keys[NUM_KEYS];
+//OW
+uint16_t s_evt_lockmask = 0;
```

Comment:
is it possible you explain what you need here?

Reply:
that's a good point (and another of the pain points I mentioned):

One wants to have button handling in widgets, since otherwise one can't achieve a "GCS-like" behavior, i.e., the widgets would just be displays but one couldn't trigger or set things through user interaction. For example, I use that to choose between video and photo mode, by offering a menu which the user can highlight, browse and select, and other such things.

My implementation is clearly a demonstration of me just not knowing how to do it better.

I later have then discovered this PR, https://github.com/opentx/opentx/pull/6688, which appears to exactly attempt the same, but from what I see along the correct way. 

I would need advice and guidance to get this straight.